dation.css

```diff
@@ -0,0 +1,703 @@
+:root {
+    scroll-behavior: smooth;
+    /* --primary-color: white;
+    --secondary-color: black; */
+    
+    /* --primary-color: #1DA1F2;
+    --secondary-color: black; */
+    
+    /* --primary-color: #1DA1F2;
+    --secondary-color: white; */
+
+    /* --primary-color: white;
+    --secondary-color: black; */
+
+    --primary-color: black;
+    --secondary-color: white;
+    
+    --shadow-color: #9CA3AF;
+    /* --glow-color: blue; */
+    --glow-color: green;
+
+    --dark-grey: #6B7280;
+    --off-white: #F3F4F6;
+    
+    /* --nav-bg-color: #1DA1F2; */
+    /* --nav-bg-color: black; */
+    --nav-bg-color: var(--primary-color);
+    --nav-txt-color: var(--secondary-color);
+
+    --padding-xs: 0.3em;
+    --padding-s: 0.7em;
+    --padding-m: 1em;
+    --padding-l: 1.3em;
+    --padding-xl: 2em;
+
+    --margin-xs: 0.3em;
+    --margin-s: 0.7em;
+    --margin-m: 1em;
+    --margin-l: 1.3em;
+    --margin-xl: 2em;
+
+    --border-radius-xs: 0.5em;
+    --border-radius-s: 1em;
+    --border-radius-m: 2em;
+    --border-radius-l: 2.5em;
+
+    --btn-bg-color: var(--primary-color);
+    /* --btn-border-color: var(--secondary-color); */
+    --btn-txt-color: var(--secondary-color);
+
+    --card-width: 280px;
+    --card-height: 420px;
+
+    --input-border-color: var(--btn-bg-color);
+    --input-focus-color: var(--dark-grey);
+
+    --selection-color: var(--dark-grey);
+}
+
+ul, li, h1, h2, h3, h4, h5, h6 {
+    margin-block-start: 0;
+    margin-block-end: 0;
+    padding-inline-start: 0;
+}
+
+h1, h2, h3, h4, h5, h6 {
+    margin-block-start: 0.1rem;
+    margin-block-end: 0.1rem;   
+}
+
+
+body {
+    margin: 0;
+    /* background: gray; */
+}
+
+.alert {
+    text-align: center;
+    position: fixed;
+    z-index: 1;
+    bottom: 0;
+    left: 0;
+    right: 0;
+    margin-bottom: var(--margin-xs);
+    margin-left: auto;
+    margin-right: auto;
+    width: 90%;
+    padding: var(--padding-xs);
+    border-radius: var(--border-radius-xs);
+    background-color: var(--nav-bg-color);
+    color: var(--nav-txt-color);
+}
+
+.avatar {
+    border-radius: 50%;
+    position: relative;
```

Comment:
Give some equal width and height here. Because if the image which you are using for avatar does not have equal width and height then the avatar will look like a perfect circle. That's why equal width and height is needed explicitly.